content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -150,7 +156,7 @@ automatically updated.
 
```

Comment:
For line 150, might want to specify where the 'Remaining hours' field is located. I had to click around till I found a SO that had this to reveal. So if it doesn't appear, someone might be lost.